packages/blockprotocol/core.d.ts

```diff
@@ -273,23 +309,45 @@ export type BlockProtocolDeleteEntityTypesFunction = {
 };
 
 // ------------------------- GENERAL / SUMMARY ----------------------------- //
+
 export type BlockProtocolFunction =
   | BlockProtocolAggregateEntitiesFunction
-  | BlockProtocolAggregateEntityTypesFunction
   | BlockProtocolCreateEntitiesFunction
-  | BlockProtocolCreateEntityTypesFunction
-  | BlockProtocolCreateLinksFunction
-  | BlockProtocolDeleteEntitiesFunction
-  | BlockProtocolDeleteEntityTypesFunction
-  | BlockProtocolDeleteLinksFunction
   | BlockProtocolGetEntitiesFunction
-  | BlockProtocolGetEntityTypesFunction
-  | BlockProtocolGetLinksFunction
   | BlockProtocolUpdateEntitiesFunction
+  | BlockProtocolDeleteEntitiesFunction
+  | BlockProtocolAggregateEntityTypesFunction
+  | BlockProtocolCreateEntityTypesFunction
+  | BlockProtocolGetEntityTypesFunction
+  | BlockProtocolDeleteEntityTypesFunction
   | BlockProtocolUpdateEntityTypesFunction
+  | BlockProtocolCreateLinksFunction
+  | BlockProtocolGetLinksFunction
   | BlockProtocolUpdateLinksFunction
+  | BlockProtocolDeleteLinksFunction
   | BlockProtocolUploadFileFunction;
```

Comment:
The original list was ordered alphabetically. I prefer sticking with this where I can to simplify visual search. I understand why you’ve regrouped the types and am happy both ways.

Reply:
I've ordered these alphabetically - I've retained the 'by target' grouping in the type below it.